client/hotrod-client/src/main/java/org/infinispan/client/hotrod/RemoteCacheManagerAdmin.java

```diff
@@ -102,4 +102,23 @@
     * @throws HotRodClientException
     */
    void reindexCache(String name) throws HotRodClientException;
+
+   /**
+    * Retrieves an existing template on the remote server cluster. If it doesn't exist, it will be created using the
+    * specified template name.
+    *
+    * @param name the name of the template to create
+    * @param configuration the configuration object
+    *
+    * @throws HotRodClientException
+    */
+   void createTemplate(String name, BasicConfiguration configuration);
```

Comment:
Why didn't you add these to CacheContainerAdmin instead so that they can be shared with EmbeddedCacheManagerAdmin

Reply:
Just an oversight, updated